apps/docs/src/guide/contracts/deploying-contracts.md

```diff
@@ -27,11 +27,23 @@ Load the contract bytecode and JSON ABI, generated from the Sway source, into th
 
 ## 4. Deploying the Contract
 
-Initialize a [`ContractFactory`](../../api/Contract/ContractFactory.md) with the bytecode, ABI, and wallet. Deploy the contract and use its methods.
+To deploy the contract we can instantiate the [`ContractFactory`](../../api/Contract/ContractFactory.md) with the bytecode, ABI, and wallet. The we can call `deployContract` method. This method returns a promise that will resolve to a `Contract` instance, ready to be used to interact with the deployed smart contract.
```

Comment:
```suggestion
To deploy the contract we can instantiate the [`ContractFactory`](../../api/Contract/ContractFactory.md) with the bytecode, ABI, and wallet. Then we can call the `deployContract` method. This method returns a promise that will resolve to a `Contract` instance which provides methods to interact with the deployed smart contract.
```

Reply:
@danielbate The docs are slightly different now https://github.com/FuelLabs/fuels-ts/pull/2597/commits/7c800e1786393467ffb92433a7a331d6ca38d4cb